tests/unit_tests/test_query_builder.py

```diff
@@ -389,3 +474,39 @@ def test_drop_field_all_regions(all_regions_query):
         "itemsPerPage",
         "total",
     ]
+
+
+def test_arguments_info_formatting(mocked_get_arguments, field_default):
+    info = field_default._arguments_info_formatter({"WAHL09": FieldMetaDict()})
```

Comment:
I find this line now became a bit confusing, as it requires the reader to remember that the empty FieldMetaDict() container class is mocked to return specific content. Alternatively one could simple inject a fixture with the proper FieldMetaDict. The latter would appear more transparent to me.